fe/fe-core/src/main/java/com/starrocks/server/OlapTableFactory.java

```diff
@@ -385,6 +386,19 @@ public Table createTable(LocalMetastore metastore, Database db, CreateTableStmt
                     if (partitionInfo instanceof ListPartitionInfo) {
                         throw new DdlException("List partition table does not support storage medium cool down currently.");
                     }
+                    if (partitionInfo instanceof RangePartitionInfo) {
+                        RangePartitionInfo rangePartitionInfo = (RangePartitionInfo) partitionInfo;
+                        List<Column> partitionColumns = rangePartitionInfo.getPartitionColumns();
+                        if (partitionColumns.size() > 1) {
+                            throw new DdlException("Multi-column range partition table " +
+                                    "does not support storage medium cool down currently.");
+                        }
+                        Column column = partitionColumns.get(0);
+                        if (!column.getType().getPrimitiveType().isDateType()) {
```

Comment:
isDateType including DATE and DATETIME, is it ok?

Reply:
yes